guides/common/modules/con_upgrade-paths.adoc

```diff
@@ -8,13 +8,8 @@ ifdef::satellite[]
 For more information, see {UpgradingPreviousDocURL}[_{UpgradingPreviousDocTitle}_] or {UpgradingDisconnectedPreviousDocURL}[_{UpgradingDisconnectedPreviousDocTitle}_].
 endif::[]
 
-.High-level upgrade steps
-
-The high-level steps in upgrading {Project} to {ProjectVersion} are as follows:
-
-. Upgrade {ProjectServer} to {ProjectVersion}.
-For more information, see xref:Project_Upgrade_Considerations_{context}[].
-. Upgrade all {SmartProxyServers} to {ProjectVersion}.
```

Comment:
When the high-level overview was formatted as a numbered list, it gave the impression that it represented a complete list of steps required for an upgrade. That was certainly not the case and I don't think we should even try to give the complete list here (that would duplicate the actual upgrade chapter). Rewording the list into a normal para a suggested below reduces the risk of misunderstanding and will make maintaining this section easier.

Reply:
I'm not sure this is better. I think the previous text tried to make it clear that you would first upgrade your `{ProjectServer}` and then you would upgrade all your proxy servers, but that maybe wasn't obvious. It may also not have been obvious that you can spread those steps out over multiple maintenance windows since there is n-1 support.

Your new text puts it in a single sentence which to me implies it is a single operation in a single maintenance window.

I think it's something like:

* Upgrade `{ProjectServer}` to `{ProjectVersion}`
  * If on EL8, at some point before upgrading to the next version: migrate to EL9
* For every `{SmartProxyServer}: find a nice maintenance window and upgrade it to `{ProjectVersion}`. Has to be completed before upgrading `{ProjectServer}` to the next version
  * If on EL8, at some point before upgrading to the next version: migrate to EL9

Now that probably isn't the most user friendly description, but I hope you at least understand it and can work it into something consumable.

Reply:
To add to this, in the resulting document you have an overview that states (for Katello):

> Review prerequisites and available upgrade paths below before upgrading your current Foreman installation to Foreman nightly.
>
> Note that you can upgrade Smart Proxies separately from Foreman. For more information, see [Upgrading Smart Proxies separately from Foreman](https://theforeman-foreman-documentation-preview-pr-3408.surge.sh/nightly/Upgrading_Project/index-katello.html#Upgrading_Proxies_Separately_from_Server_upgrading-connected).

Then there's an Upgrade paths section. That last sentence is something that IMHO belongs in the Upgrade paths section, but then the overview is very short. Should they be combined?

Reply:
Well, the whole thing makes a lot more sense to me now. Thanks! I'll keep working on it.

Reply:
> To add to this, in the resulting document you have an overview that states (for Katello):
> 
> > Review prerequisites and available upgrade paths below before upgrading your current Foreman installation to Foreman nightly.
> > Note that you can upgrade Smart Proxies separately from Foreman. For more information, see [Upgrading Smart Proxies separately from Foreman](https://theforeman-foreman-documentation-preview-pr-3408.surge.sh/nightly/Upgrading_Project/index-katello.html#Upgrading_Proxies_Separately_from_Server_upgrading-connected).
> 
> Then there's an Upgrade paths section. That last sentence is something that IMHO belongs in the Upgrade paths section, but then the overview is very short. Should they be combined?

Yes. Similarly, there is a prerequisite a bit further along that describes how long the upgrade takes and that users need to properly plan for it. That also seems closely related and might work better together with the other related information in Upgrade paths.